include/boost/geometry/algorithms/discrete_frechet_distance.hpp

```diff
@@ -153,19 +153,35 @@ inline typename distance_result
             typename point_type<Geometry2>::type,
             Strategy
         >::type
-frechet_distance(Geometry1 const& g1, Geometry2 const& g2, Strategy const& strategy)
+frechet_distance(Geometry1 const& geometry1, Geometry2 const& geometry2, Strategy const& strategy)
 {
-    return dispatch::frechet_distance<Geometry1, Geometry2>::apply(g1, g2, strategy);
+    return dispatch::frechet_distance<Geometry1, Geometry2>::apply(geometry1, geometry2, strategy);
 }
 
 // Algorithm overload using default Pt-Pt distance strategy
+/*!
+\brief calculate discrete frechet distance between two geometries
+\ingroup discrete_frechet_distance
+\tparam Geometry1 \tparam_geometry
+\tparam Geometry2 \tparam_geometry
+\param geometry1 Input geometry
+\param geometry2 Input geometry
+
+\qbk{[include reference/algorithms/discrete_frechet_distance.qbk]}
+ 
+\qbk{
+[/heading Example]
+[/discrete_frechet_distance]
+[/discrete_frechet_distance_output]
+}
+*/
```

Comment:
Great, it's almost done. You also need the same-style description above the other variant of the algorithm taking Strategy with `\qbk{distinguish,with strategy}`.

Reply:
Regarding the *Available Strategies* you could take them from here: https://github.com/boostorg/geometry/blob/develop/include/boost/geometry/algorithms/detail/distance/interface.hpp#L340

you only need:

    \* [link geometry.reference.strategies.strategy_distance_pythagoras Pythagoras (cartesian)]
    \* [link geometry.reference.strategies.strategy_distance_haversine Haversine (spherical)]

Just have in mind that you need to have a newline between *Available strategies* and *Examples* like here: https://github.com/boostorg/geometry/blob/develop/include/boost/geometry/algorithms/densify.hpp#L369